Tests/KumoTests/Fixtures/Common/NetworkTest.swift

```diff
@@ -52,9 +52,40 @@ class NetworkTest: XCTestCase {
                     XCTFail("Expectation violated - test '\(function)' emitted an element: \(element).", file: file, line: line)
                     expect.fulfill()
                 })
-                    .withLifetime(of: self)
+                .withLifetime(of: self)
                 self.wait(for: [expect], timeout: 10)
             }
         }
     }
+    
+    func perform<T: Decodable, Method: _RequestMethod, Resource: _RequestResource, Parameters: _RequestParameters, Body: _RequestBody, Key: _ResponseNestedKey>(_ request: HTTP._Request<Method, Resource, Parameters, Body, Key>) -> AnyPublisher<T, Error> {
+        Deferred<AnyPublisher<T, Error>> {
+            Future<T, Error> { [self] promise in
+                Task {
+                    do {
+                        let result: T = try await service.perform(request)
+                        promise(.success(result))
+                    } catch let error {
+                        promise(.failure(error))
+                    }
+                }
+            }.eraseToAnyPublisher()
+        }.eraseToAnyPublisher()
+    }
+    
+    func performs<Method: _RequestMethod, Resource: _RequestResource, Parameters: _RequestParameters, Body: _RequestBody, Key: _ResponseNestedKey>(_ request: HTTP
```

Comment:
I'm confused by the naming between `perform` vs `performs` and when I'd use one vs the other?